src/PlaywrightSharp.Chromium/ChromiumBrowserType.cs

```diff
@@ -1,17 +1,30 @@
 using System;
 using System.Collections.Generic;
+using System.IO;
+using System.Runtime.InteropServices;
 using System.Threading.Tasks;
 
 namespace PlaywrightSharp.Chromium
 {
     /// <inheritdoc cref="IBrowserType"/>
     public class ChromiumBrowserType : IBrowserType
     {
+        private readonly string _revision = "733125";

```

Comment:
```suggestion
        private static readonly string _revision = "733125";
```

Reply:
Playwright has that as an instance prop but I think we can make it static due to our code nature.